### PR TITLE
[HSEARCH] 7.1.0.Alpha2

### PR DESCRIPTION
**Pull request by marko-bekhta**

### CODE REVIEW

File: posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+:awestruct-layout: blog-post
+:hsearch-doc-url-prefix: https://docs.jboss.org/hibernate/search/7.1/reference/en-US/html_single/
+:hsearch-jira-url-prefix: https://hibernate.atlassian.net/browse
+:hsearch-version-family: 7.1
+:hsearch-jira-project-id: 10061
+:hsearch-jira-version-id: 32220
+---
+
+We just published Hibernate Search 7.1.0.Alpha2,
+a second alpha release of the next major version of Hibernate Search.
```
Comment (yrodiere) [inline posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc]:
```suggestion
a second alpha release of the next minor version of Hibernate Search.
```

File: posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+Which beyond other improvements brings better performance for vector search.
+[[elasticsearch-version]]
+Elasticsearch (link:{hsearch-jira-url-prefix}/HSEARCH-5032[HSEARCH-5032]::
+The Elasticsearch backend now is also compatible with Elasticsearch 8.12 as well as with other versions that were already compatible.
+[[others-version]]
+Others::
+* link:{hsearch-jira-url-prefix}/HSEARCH-5032[HSEARCH-5032]: Upgrade to Elasticsearch **client** 8.12.0
+* link:{hsearch-jira-url-prefix}/HSEARCH-5057[HSEARCH-5057]: Upgrade to AWS SDK 2.23.3
+* link:{hsearch-jira-url-prefix}/HSEARCH-5047[HSEARCH-5047]: Upgrade to JBeret 2.2.0.Final
+* link:{hsearch-jira-url-prefix}/HSEARCH-5057[HSEARCH-5057]: Upgrade to Upgrade AWS SDK to 2.23.3
```
Comment (yrodiere) [inline posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc]:
```suggestion
* link:{hsearch-jira-url-prefix}/HSEARCH-5057[HSEARCH-5057]: Upgrade to AWS SDK 2.23.3
* link:{hsearch-jira-url-prefix}/HSEARCH-5047[HSEARCH-5047]: Upgrade to JBeret 2.2.0.Final
```

**Review by yrodiere — COMMENTED** [review]
Thanks, just a few suggestions.

File: posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc (hunk 2 above)
Comment (marko-bekhta) [reply posts/MarkoB/2024-01-24-hibernate-search-7-1-0-Alpha2.adoc]: 🙈  thanks!!!

**Pull request merged**